konfetti/src/main/java/nl/dionsegijn/konfetti/ParticleSystem.kt

```diff
@@ -153,8 +153,13 @@ class ParticleSystem(private val konfettiView: KonfettiView) {
         return this
     }
 
+    fun setRotationSpeedMultiplier(multiplier: Float): ParticleSystem {
```

Comment:
Could you add documentation to this function detailing what it does like the other functions?